tests/test_apikey_api.py

```diff
@@ -0,0 +1,141 @@
+import json
+import uuid
+
+import httpretty
+from mock import patch
+
+from examples import misc_examples as misc_ex
+from lecli.api_key import api
+
+
+@httpretty.activate
+@patch('lecli.api_utils.get_account_resource_id')
+@patch('lecli.api_utils.get_rw_apikey')
+@patch('lecli.api_key.api._url')
+def test_get_api_keys(mocked_url, mocked_rw_apikey, mocked_account_resource_id, capsys):
+    mocked_url.return_value = '', misc_ex.MOCK_API_KEY_URL
+    mocked_rw_apikey.return_value = uuid.uuid4()
+    mocked_account_resource_id.return_value = str(uuid.uuid4())
+    httpretty.register_uri(httpretty.GET, misc_ex.MOCK_API_KEY_URL,
+                           status=200,
+                           content_type='application/json',
+                           body=json.dumps({}))
+
+    api.list()
+
+    out, err = capsys.readouterr()
+    assert not err
```

Comment:
Can we check that something has been printed to `out`?
If someone creates a branch with no error output, it will still pass